packages/components/affix/__tests__/affix.test.tsx

```diff
@@ -1,8 +1,7 @@
 import { nextTick } from 'vue'
 import { mount } from '@vue/test-utils'
 import { afterAll, beforeAll, describe, expect, test, vi } from 'vitest'
-import defineGetter from '@element-plus/test-utils/define-getter'
-import makeScroll from '@element-plus/test-utils/make-scroll'
+import { defineGetter, makeScroll } from '@element-plus/test-utils'
```

Comment:
Thank you for your contribution, but in my opinion it might be clearer to show the specific file name of the import.

Reply:
> Thank you for your contribution, but in my opinion it might be clearer to show the specific file name of the import.

I agree with you.
However, just like using `@element-plus/hooks`, my first thought was to use module redirection(`import { sleep } from '@element-plus/test-utils'`), In fact, I don't get any errors when i use, but throws error when I run. 

Maybe we can just fix the export problem in pkg.json, and without adjusting the existing import.
<img width="708" alt="image" src="https://github.com/element-plus/element-plus/assets/69044080/ef3655db-43a6-4301-8108-c52ae3d4b009">

Reply:
Yes, there should be some adjustments here.